boards/BMS/Test/Src/Test_StateMachine.cpp

```diff
@@ -813,4 +813,39 @@ TEST_F(BmsStateMachineTest, check_cell_monitors_can_signals_in_all_states)
         CANMSGS_BMS_NON_CRITICAL_ERRORS_CELL_MONITOR_5_DIE_TEMP_OUT_OF_RANGE_OVERFLOW_CHOICE);
 }
 
```

Comment:
```suggestion
// BMS-12
```
It seems both tests fall under BMS-12